src/components/Sidebar.js

```diff
@@ -1,41 +1,75 @@
-import React from 'react';
+import React, { Component } from 'react'
 import { withStyles } from 'material-ui/styles';
 
 import LogoutLink from './LogoutLink';
 import Divider from 'material-ui/Divider';
 import Toolbar from 'material-ui/Toolbar';
-import List from 'material-ui/List';
+import { List, ListItem } from 'material-ui';
 import SidebarNavLink from './SidebarNavLink';
 import Logo from '../images/cambridgeuniversity_logo.svg';
+import {connect} from "react-redux";
+import {getSelf} from "../redux/actions/lookupApi";
 
 const styles = theme => ({
   drawerHeader: theme.mixins.toolbar,
   nested: { paddingLeft: theme.spacing.unit * 4 },
   camLogo: { width: '145px', paddingTop: '10px'},
-  tagLine: { fontSize: 12},
-  logoToolbar: { flexDirection:'column', alignItems: 'flex-start', paddingLeft: theme.spacing.unit * 2  }
+  tagLine: { fontSize: 12 },
+  logoToolbar: { flexDirection:'column', alignItems: 'flex-start', paddingLeft: theme.spacing.unit * 2 },
+  assetHeading: { padding: '12px 16px' }
 });
 
 /**
  * The content of the IAR application side bar.
  */
-const Sidebar = ({ classes, history, logout }) => (
-  <div>
-    <div className={classes.drawerHeader}>
-      <Toolbar className={classes.logoToolbar} disableGutters={true}>
-        <img src={Logo} className={classes.camLogo} alt="Cambridge University Logo"/>
-        <div className={classes.tagLine}>Information Asset Register</div>
-      </Toolbar>
+class Sidebar extends Component {
+
+  /**
+   * If we are signed in and we haven't retrieved the profile - then retrieve the profile.
+   * TODO I guess we need to find a better place for this as Sidebar won't always be rendered.
```

Comment:
This could be a "FetchSelf" component which renders to null but fetches the self profile on successful login. (See, e.g. https://github.com/rjw57/iar-frontend/blob/9de0cce8e60806f03dd70bfb68623dc2bedf59db/src/components/FetchLookupInstitutions.js and its use in https://github.com/uisautomation/iar-frontend/pull/70/commits/9de0cce8e60806f03dd70bfb68623dc2bedf59db#diff-4495a4a3bf464d8e86c545b68c5783a6.)

Another way would be to have some redux middleware which notices a successful login and automatically dispatches the get self action.

The former has the advantage that the fetch is explicitly shown in the component hierarchy. The latter has the advantage that it happens "by magic" to the rest of the app. I'm not sure yet what I'd consider "best". I chose the former for the Lookup institution fetching but could be persuaded to the latter.

For the moment, though, and if you agree, I think this is merge-able as is. We can move the decision to the next story which makes use of self.